WDL.YAML-tmLanguage

```diff
@@ -106,4 +108,14 @@ repository:
       patterns:
       - include: '#constant_placeholder'
       - include: '#escaped_char'
+
+  command_block:
+    patterns:
+    - comment: command block
+      name: command.block.wdl
+      begin: (command)\s\{\n
```

Comment:
I guess that this should probably be `\s*`? To cover `command{` and `command     {`, etc

Reply:
I wasn't sure if command{ was proper syntax, I'll fix.